stories/2022-03-27_Glass_ML/ML_20th_Century.ipynb

```diff
@@ -0,0 +1,2194 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from sklearn.metrics import confusion_matrix
</pre><p>Jennifer Ding:</p><ul><li>Clean up imports in Cell 13, you can do something like</li></ul><p>from sklearn.metrics import confusion_matrix, mean_absolute_error, cohen_kappa_score</p><p>Joe Cerniglia:</p><p>That is an excellent idea. I will do that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>